Dockerfile

```
@@ -0,0 +1,26 @@
+# Start with Python base image
+FROM python:3.9-slim
```

```suggestion
FROM python:3.12-alpine
```
Switch to a more recent version of Python.
3.12 is the latest one supported by wallycore.
Also Python images of the alpine variant are about half the size of the slim images.